spec/src/modules/tracker.js

```diff
@@ -891,7 +895,13 @@ describe('ConstructorIO - Tracker', () => {
         fetch: fetchSpy,
       });
 
-      expect(tracker.trackRecommendationClickThrough({})).to.equal(true);
+      expect(tracker.trackRecommendationClick({
+        result_count: parameters.result_count,
+        result_page: parameters.result_page,
+        result_id: parameters.result_id,
+        pod_id: parameters.pod_id,
+        strategy_id: parameters.strategy_id,
+      })).to.equal(true);
```

Comment:
These parameters (minus `section`) should have been passed previously - this is a correction.